### DISCUSSION TITLE
Convert my ventoy pendrive to an ISO to boot a vm from it

### DISCUSSION THREAD

**Discussion by sebma** [ventoy/Ventoy#2323] [Q&A]
Hi, I'd like to boot a vmware vm from an ISO image of my ventoy pendrive, how can I convert it to an ISO file ?

**Comment by 0hip** [comment]
https://www.ventoy.net/en/doc_livecd.html
2. LiveCD User
1. Ventoy LiveCD is only for Windows users, and only needed when Ventoy2Disk.exe always fail.
2. Linux users can directly use linux version of Ventoy, and no need to use this LiveCD.
**3. If you don't want to use USB drive, you can also test Ventoy with Virtual Machine. Directly use the LiveCD ISO and install Ventoy to the disk inside the VM.**

**Reply by sebma** [reply]
@0hip Hi, my question was a little confusing so I just updated it.
I need to boot a VM from an ISO image file of my Ventoy pendrive.
How can I convert my Ventoy pendrive to an ISO image file ?

**Reply by 0hip** [reply]
https://www.tecmint.com/create-an-iso-from-a-bootable-usb-in-linux/
After you get the ISO image of your Ventoy pendrive, is your next step to copy it to a ventoy pendrive?

**Comment by steve6375** [comment]
You can't boot ventoy from an iso. Instead convert it to a fixed vhd file.
The vhd can then be loaded into the VM attached as a hard disk

**Reply by sebma** [reply]
@steve6375 Great, or to a VMDK image :smile: Then, I can add ISO files into the remaining space of the VMDK, right ?
